src/spin.ts

```diff
@@ -15,7 +18,8 @@ export async function install(version: string): Promise<void> {
     }
 
     // remove 'v' from version before verifying
-    if (result.stdout.indexOf(version.replace(/^v/, '')) === -1) {
+    // if canary version, ignore the version check
```

Comment:
Can't leave a comment on the previous line, but I'm not quite sure why we're doing a version check at all.  What is the circumstance under which we fear that downloading a particular version would actually get a different version?

Reply:
was just trying to be helpful here by doing a sanity check on version. may be I should remove it.

Reply:
How about skipping the check, but running `spin --version` to print to the logs, so the user has a diagnostic if something goes wrong?  (And can trace back to an actual version if they asked for canary or latest.)